### PR TITLE
Fix congress message actions

### LINKED ISSUE #650: Add Error Message to Congress Message Action
For a congress message action, the tool breaks with no error message when an address is misentered or no reps are found
Screen Shot 2019-08-28 at 2 48 26 PM

### PR DESCRIPTION
**Pull request by esoterik**
closes #650

### CODE REVIEW

**Comment by esoterik** [comment]
This also corrects the order of the forms, as it should be Find Reps -> Additional Info -> Customize Message

File: app/views/congress_messages/_form.html.erb [hunk 1]
```diff
@@ -40,9 +40,9 @@
   <%= hidden_field_tag :bioguide_ids, @message.targets %>
 
   <div class="submit-wrapper">
-    <input type="submit" value="Submit" class="btn action">
+    <input id="to-page-3" type="submit" value="Next >" class="btn action">
     <% if current_user && current_user.privileged_role? %>
-      <input type="submit" value="Test" name="test" class="btn action test">
+      <input id="to-page-3" type="submit" value="Test" name="test" class="btn action test">
```
Comment (lschatzkin) [inline app/views/congress_messages/_form.html.erb]: Can we make these class="eff-button" ? I'm trying to get rid of those bootstrap buttons.

**Comment by lschatzkin** [comment]
Step 3 never appears for me, I just end up at the thank you with progress numbers in the wrong sequence.
Screenshot from 2019-09-06 13-23-31

**Comment by esoterik** [comment]
also closes #689!

**Comment by esoterik** [comment]
ready for re-review :smiley_cat:

**Comment by lschatzkin** [comment]
Hmm, I'm still not having the same issue i.e. customize messages page is not appearing, I just go straight to thank you page with the incorrect progress numbers.

**Comment by lschatzkin** [comment]
YES! This is working.
One tiny 
message-input
problem: there seems to be an errant message input field in the Common Fields.

**Comment by esoterik** [comment]
unfortunately that's a separate congress forms bug -- it made the assumption that the custom message would be written first. mind if i open up a separate ticket for that?

**Comment by lschatzkin** [comment]
Sure, I will go ahead and merge this and you can work on that issue. If it's too bothersome, we can just hide it with css.

**Pull request merged**